specs/src/specs/data_structures.md

```diff
@@ -498,16 +498,20 @@ For shares **with a reserved namespace ID through [`NAMESPACE_ID_MAX_RESERVED`](
 > **Note** The first [`NAMESPACE_ID_BYTES`](./consensus.md#constants) of a share's raw data `rawData` is the namespace ID of that share, `namespaceID`. The next [`SHARE_INFO_BYTES`](./consensus.md#constants) bytes are for share information.
 
 - If this is the first share of a sequence, the next 1 to 10 bytes contain a [varint](https://developers.google.com/protocol-buffers/docs/encoding) of the `uint64` length of the sequence that follows, in bytes.
-- The next [`SHARE_RESERVED_BYTES`](./consensus.md#constants) bytes is the starting byte of the length of the [canonically serialized](#serialization) first request that starts in the share, or `0` if there is none, as a one-byte big-endian unsigned integer (i.e. canonical serialization is not used). In the example below, with a share size of `256` the reserved byte would be `80` (or `0x50` in hex).
+- The next [`SHARE_RESERVED_BYTES`](./consensus.md#constants) bytes are the starting byte of the length of the [canonically serialized](#serialization) first request that starts in the share, or `0` if there is none, as an unsigned [varint](https://developers.google.com/protocol-buffers/docs/encoding).
 - The remaining [`SHARE_SIZE`](./consensus.md#constants)`-`[`NAMESPACE_ID_BYTES`](./consensus.md#constants)`-`[`SHARE_INFO_BYTES`](./consensus.md#constants) `-` 1 to 10 bytes (if this is the first share of a sequence) `-` [`SHARE_RESERVED_BYTES`](./consensus.md#constants) bytes are transactions, intermediate state roots, or evidence data depending on the namespace of ths share. Each transaction, intermediate state root, or evidence is prefixed with a [varint](https://developers.google.com/protocol-buffers/docs/encoding) of the length of that unit.
 - If there is insufficient transaction, intermediate state root, or evidence data to fill the share, the remaining bytes are filled with `0`.
 
-First share in a sequence:
+First share in a sequence:<br>
 ![fig: compact start share.](./figures/compact_start_share.svg)
 
-Continuation share in a sequence:
+where reserved bytes would be `14` encoded as an unsigned varint (`[0b1110, 0b0]`).
```

Comment:
These are weird because you're truncating leading zeroes. Would prefer making these all 8 bits. Also, are you sure this is correct? With varint, if it takes more than one byte then the MSB is one.

Reply:
> Would prefer making these all 8 bits

Agreed, will fix in an upcoming commit.

> Also, are you sure this is correct? With varint, if it takes more than one byte then the MSB is one.

Yes because `14` doesn't occupy more than one byte so the most significant bit is off for the first byte and `0` for the second byte. Ref: https://go.dev/play/p/TqP2QrcYh6z

<img width="478" alt="Screenshot 2022-11-13 at 9 49 21 PM" src="https://user-images.githubusercontent.com/3699047/201564803-08ab16b2-e71e-487f-b5f9-b7007877936a.png">

Reply:
Resolved by https://github.com/celestiaorg/celestia-app/pull/939/commits/fae99b678722ee8ddf15a36a4766c1b64dee6f25